repair_type_sequence/views/repair_type_views.xml

```diff
@@ -7,14 +7,12 @@
         <field name="arch" type="xml">
             <xpath expr="//sheet" position="inside">
                 <group>
-                    <field
-                        name="sequence_prefix"
-                        attrs="{'readonly': [('sequence_id', '!=', False)]}"
-                    />
+                    <field name="sequence_prefix" readonly="sequence_id != False" />
                     <field
                         name="sequence_number_next"
                         style="padding-right: 1.0em"
-                        attrs="{'readonly': [('sequence_id', '!=', False)], 'invisible': [('sequence_prefix', '=', False)]}"
+                        readonly="sequence_id != False"
+                        invisible="sequence_prefix == False"
```

Comment:
More readable with:
`
readonly="sequence_id"
invisible="not sequence_prefix"
`